Makefile

```diff
@@ -194,3 +194,15 @@ domains-apply: domains-init ## terraform apply for dns resources
 
 domains-destroy: domains-init ## terraform destroy for dns resources
 	terraform -chdir=terraform/domains/environment_domains destroy -var-file config/$(CONFIG).tfvars.json
+
+test-cluster:
+	$(eval CLUSTER_RESOURCE_GROUP_NAME=s189t01-tsc-ts-rg)
+	$(eval CLUSTER_NAME=s189t01-tsc-test-aks)
+
+production-cluster:
+	$(eval CLUSTER_RESOURCE_GROUP_NAME=s189p01-tsc-pd-rg)
+	$(eval CLUSTER_NAME=s189p01-tsc-production-aks)
```

Comment:
Having the values hard coded in the Makefile means we need to keep these in sync with the cluster data in the Terraform modules. Can we instead use `terraform` to get these values like we do here:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/ec537b9518625b0d3039715936d0e6ba65988e3c/.github/actions/set-kubernetes-credentials/action.yaml#L37-L38

Reply:
The output is only present if you run terraform init, which requires a PIM request and the Contributor role. This is a security risk as the role has read/write access to all the azure resources of all the services.

We want to simplify and increase security and let the devs access AKS without elevating to Contributor. This will prevent running terraform by default. Unfortunately this causes a little bit of duplication.

Reply:
Okay fair enough. I think we can remove those outputs as they were only used for this.

Reply:
We can also remove the outputs themselves as they were only used in the `set-kubernetes-credentials` action:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/cb1b9de3b42699c489313c6cac7d21620df54d73/terraform/application/outputs.tf#L13-L19

Reply:
removed